tools/settings_template.py

```diff
@@ -33,7 +33,8 @@
 
 JAVA = 'java' # executable
 
-# CLOSURE_COMPILER = '..' # define this to not use the bundled version
+# To install Closure compiler, run 'npm install' in Emscripten root directory
+CLOSURE_COMPILER = '{{{ CLOSURE_COMPILER }}}'
```

Comment:
Should we just remove this line completely from the config?  Do we want to support non-standard locations/configurations for closure compiler?  I'm tempted to say no.

Reply:
I don't want to break existing users. No idea how that came to be, but I'll leave that for later, if someone wants to drop this. (no idea if someone might break if that is removed, but I think that's for separate topic)

Reply:
What about leaving this alone then?   Any reason not to leave it as a comment?   Seems like this line could just be reverted.